subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -331,7 +331,7 @@ This way when the task's results are loaded from cache, the next execution can a
 
 However, if the state files are non-relocatable, then they can't be shared via the build cache.
 Indeed, when the task is loaded from cache, any such state files must be cleaned up to prevent stale state from confusing the tool during the next execution.
-Gradle can ensure such stale files are removed if they are declared via `task.localState.register()` or if a property is marked with the `@LocalState` annotation.
+Gradle can ensure such stale files are removed if they are declared via `link:{javadocPath}/org/gradle/api/tasks/TaskLocalState.html#register-java.lang.Object++...++-[task.localState.register()]` or if a property is marked with the `@link:{javadocPath}/org/gradle/api/tasks/LocalState.html[LocalState]` annotation.
```

Comment:
Just an FYI: you only need to put the backticks around the text you want monospaced. For example:

> link:{javadocPath}/org/...[`task.localState.register()`]

Also, not sure why you don't include the '@' within the link.

And a final note: the user manual currently restyles all links to the Javadoc API and DSL reference as monospace, regardless of the backticks. However, I like to keep the backticks because I eventually want to roll back that CSS/JS hack because it restyles these links even if the link text isn't a class/method/other bit of code.

Reply:
When I added the backticks in the `[]`, then it was not apparent anymore that the text is a link, at least when I generated the userguide locally. Is that preferred or do we want to keep the blue link color?

Reply:
Hmm...I haven't seen that happen. Or maybe I have but it hasn't registered. Are you testing with a file:// URL or running a local HTTP server?

Reply:
In fact, other API links in that chapter don't have backticks at all (relying on the CSS styling)

Reply:
This is what it looks like when using
```
Gradle can ensure such stale files are removed if they are declared via link:{javadocPath}/org/gradle/api/tasks/TaskLocalState.html#register-java.lang.Object++...++-[`task.localState.register()`] or if a property is marked with the link:{javadocPath}/org/gradle/api/tasks/LocalState.html[`@LocalState`] annotation.
```

![image](https://user-images.githubusercontent.com/423186/55940741-03ecf000-5c41-11e9-8e42-2a86f6285092.png)

I am using a local webserver (the one built into IDEA).

Reply:
OK. Does it appear in monospace if you don't have any backticks at all? I'm intrigued now. Not behaving as I expected.

Reply:
Yes: 
![image](https://user-images.githubusercontent.com/423186/55941579-c1c4ae00-5c42-11e9-879f-bba2cc5a1e23.png)

Reply:
And with the backticks, there is a grey background: 
![image](https://user-images.githubusercontent.com/423186/55941688-ffc1d200-5c42-11e9-90f9-97e544b8837e.png)